src/components/widgets/filesystem/FileEditorDialog.vue

```diff
@@ -57,6 +57,19 @@
             </v-icon>
             <span v-if="!$vuetify.breakpoint.smAndDown">{{ $t('app.general.btn.config_reference') }}</span>
           </app-btn>
+          <app-btn
+            v-if="!printerPrinting && configMap.link"
```

Comment:
The structure toggle button is hidden while printing, I don't believe that should be the case

```suggestion
            v-if="configMap.link"
```